plugins/myanimelist.rb

```diff
@@ -0,0 +1,63 @@
+#  -*- coding: utf-8 -*-
+#  myanimelist.rb
+#  Author: Winston Weinert
+#  ------------------------
+#  A Cinch plugin that provides MyAnimeList search interaction for yossarian-bot.
+#  ------------------------
+#  This code is licensed by Winston Weinert under the MIT License.
+#  http://opensource.org/licenses/MIT
+
+require 'htmlentities'
+require 'myanimelist'
+
+require_relative 'yossarian_plugin'
+
+class MyAnimeListSearch < YossarianPlugin
+  include Cinch::Plugin
+  use_blacklist
+
+  def initialize(*args)
+    super
+    MyAnimeList.configure do |cfg|
+      cfg.username = ENV['MAL_USERNAME']
+      cfg.password = ENV['MAL_PASSWORD']
+    end
+    @entities = HTMLEntities.new
+  end
+
+  def usage
+    '!<anime|manga> <search> - Search MyAnimeList for anime or manga.'
```

Comment:
Instead of `!anime` and `!manga`, what do you think about just `!mal`? It looks like both use the same method call/API endpoint anyways.

Reply:
It appears to call different paths - https://myanimelist.net/modules.php?go=api#animemangasearch https://github.com/harveyico/myanimelist/blob/master/lib/myanimelist/api.rb

Reply:
Oh, I didn't see the conditional in `search`. Never mind then!